Stripe/STPCard.m

```diff
@@ -135,6 +137,54 @@ - (BOOL)isEqualToCard:(STPCard *)other {
     return [self.cardId isEqualToString:other.cardId];
 }
 
+#pragma mark - Description
+
+- (NSString *)description {
+    NSString *fundingDescription;
+
+    switch (self.funding) {
```

Comment:
nit: can we pull this out into a `stringFromFunding` method, and refactor `fundingFromString` so that we derive the outputs of both the function and its inverse from the same dictionary? [0]

I know the conversion is kinda tedious and we don't use the pattern super consistently, but I generally like coupling inverse functions like these so they don't drift apart

https://github.com/stripe/stripe-ios/blob/master/Stripe/STPSource.m#L42-L67